datasets/DEFRA-ENV01-Emissions-of-air-pollutants/main.py

```diff
@@ -0,0 +1,203 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.10.2
+#   kernelspec:
+#     display_name: Python 3
+#     language: python
+#     name: python3
+# ---
+
+# DEFRA-ENVO1-Emmissions of air pollutants
+
+import json
+import pandas as pd
+from gssutils import *
+
+cubes = Cubes('info.json')
+
+info = json.load(open('info.json'))
+landingPage = info['landingPage']
+landingPage
+
+metadata = Scraper(seed='info.json')
+
+distribution = metadata.distribution(latest=True)
+
+tabs = distribution.as_databaker()
+print(len(tabs), ' ', [tab.name for tab in tabs])
+
+contents = tabs[1]
+titles = contents.excel_ref('B9').expand(DOWN) - contents.excel_ref('B22').expand(DOWN)
+title_list = [title.value for title in titles]
+print(*title_list, sep ='\n')
+print(len(title_list))
```

Comment:
print statements can be removed in final code - great for stage one though (I need to get in the better practice of removing them)